### PR TITLE
🔀 Add eslint prettier & vue plugins

### PR DESCRIPTION
**Pull request by inDream**
Connects #34 
Connects #43

### CODE REVIEW

**Comment by nwingt** [comment]
Besides, seems prettier has conflict with CSScomb setting

**Comment by nwingt** [comment]
Maybe try [stylelint.io](https://stylelint.io)

File: .eslintrc.js [hunk 1]
```diff
@@ -9,20 +13,44 @@ module.exports = {
...
+    'prettier/prettier': ['error'],
+    'vue/attributes-order': [
+      2,
+      {
+        order: [
+          'LIST_RENDERING',
+          'CONDITIONALS',
+          'RENDER_MODIFIERS',
+          'UNIQUE',
+          'GLOBAL',
```
Comment (nwingt) [inline .eslintrc.js]: One second thought, please swap `UNIQUE` and `GLOBAL` 🙏

**Comment by inDream** [comment]
@nwingt stylelint has stylelint-order, any preferred rules config?
https://github.com/hudochenkov/stylelint-order

File: .eslintrc.js [hunk 2]
```diff
@@ -9,20 +13,52 @@ module.exports = {
...
+    ],
+    'node/no-unsupported-features': [
+      'error',
+      {
+        ignores: ['modules'],
+      },
+    ],
+    'prettier/prettier': ['error'],
+    'vue/attributes-order': [
+      2,
```
Comment (nwingt) [inline .eslintrc.js]: Please change 2 to 'error'

**Comment by inDream** [comment]
@nwingt updated thanks

**Pull request merged**